package.json

```diff
@@ -49,7 +49,8 @@
     "stable": "^0.1.5",
     "strong-remoting": "^2.13.2",
     "uid2": "0.0.3",
-    "underscore.string": "~2.3.3"
+    "underscore.string": "~2.3.3",
+    "depd": "~1.0.0"
```

Comment:
Please use `^1.0.0` so that we are automatically picking up new compatible versions in the future.